subprojects/docs/src/docs/userguide/javaLibraryPlugin.adoc

```diff
@@ -55,6 +55,12 @@ Dependencies appearing in the `api` configurations will be transitively exposed
 The `compile` configuration still exists but should not be used as it will not offer the guarantees that the `api` and `implementation` configurations provide.
 ====
 
+If your build consumes a published module with POM metadata, the Java and Java Library plugins both honor api and implementation separation through the scopes used in the pom. Meaning that the compile classpath only includes `compile` scoped dependencies, while the runtime classpath adds the `runtime` scoped dependencies as well. While this often does not have an effect for modules published with Maven, it can have a big impact for modules developed with the Java Library plugin and published with Gradle.
+
+[NOTE]
+====
+Separating compile and runtime scope of modules is active by default in Gradle 5.0+. In Gradle 4.6+, you need to activate it by by setting `org.gradle.advancedpomsupport=true` in gradle.properties.
```

Comment:
minor typo: duplicate "by"

Applies to all added instance of that note

Reply:
Fixed in all the palces.